HTTPServerProject/HTTPServerProxy/Response.cs

```diff
@@ -0,0 +1,135 @@
+using HTTPServerRead.Streams;
+using HTTPServerRead.Header;
+using HTTPServerRead.Body;
+using HTTPServerWrite.Response;
+using HTTPServerWrite.Streams;
+
+namespace HTTPServerProxy.Response;
+
+public class ProxyResponse
+{
+    Header _header;
+    Body _body;
+    IReadStreams _reader;
+    IWriteStreams _writer;
+    string _path;
+    string _type;
+
+    public ProxyResponse(IReadStreams reader, IWriteStreams writer, string path, string type)
+    {
+        _reader = reader;
+        _writer = writer;
+        _header = new Header(_reader);
+        _body = new Body(_reader);
+        _path = path;
+        _type = type;
+    }
+
+    public void GetResponse()
+    {
+        var code = GetStatusCode();
+        var headers = GetHeaders();
+        var body = GetBody();
+        var status = StatusWrapper(_type);
+        
+        if (_type == "DELETE")
+        {
+            var response = new WriteResponse(_writer, status);
+            response.GetResponse();
+        }
+        else
```

Comment:
You can shrink this up with an `else if (code == 200)`

Reply:
Agreed.